### PR TITLE
Modified the send_missing_surveys script to be run again after fixes …

### PR DESCRIPTION
**Pull request by aoby**
…in https://github.com/code-dot-org/code-dot-org/pull/10007

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Given that `bin/oneoff/` is meant to be a history of scripts, does it make sense to put this into a new file?

File: bin/oneoff/send_missing_pd_workshop_surveys [hunk 1]
```diff
@@ -9,45 +9,61 @@ require src_dir 'database'
...
+
+  # Find all surveys sent since July, when a prior version of this script was run,
+  # and make sure their associated enrollment.survey_sent_at field is set.
+  count = 0
+  deliveries = DB[:poste_deliveries].where(message_id: dashboard_message_id).
+    where('sent_at > "2016-07-01"').
+    where(Sequel.like(:params, '%Please take a minute to complete this short survey%')).
+    where(Sequel.like(:params, "%pd-workshop-survey%"))
+
+  deliveries.each do |delivery|
```
Comment (ashercodeorg) [inline bin/oneoff/send_missing_pd_workshop_surveys:21]: How many of these are there? Should the iteration be batched?

**Comment by ashercodeorg** [comment]
LGTM (beyond comments).

File: bin/oneoff/send_missing_pd_workshop_surveys (hunk 1 above)
Comment (aoby) [inline bin/oneoff/send_missing_pd_workshop_surveys:21]: Only 1827 as of right now, so probably not necessary.

**Comment by aoby** [comment]
re: new file - yeah I thought about that. I could go either way. We still have everything archived in git, and if we need to easily refer to this script in the future it would make sense to use the newer version (whether it's this updated file or a new one).

I don't have a strong preference, happy to move it into a new script if you prefer.

**Comment by jeremydstone** [comment]
LGTM

**Pull request merged**